Twittimation/HttpRequest.cs

```diff
@@ -0,0 +1,196 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Twittimation
+{
+    public class HttpRequest
```

Comment:
For reference, it's generally significantly better and more performant to use a single-instance `HttpClient` instead of `HttpWebRequest`.

No need to change that for this PR.